bodybank/character_generator/views.py

```diff
@@ -14,12 +19,38 @@ def get_ego(request):
             ego = form.save(commit=False)
             ego.user = request.user
             ego.save()
-            return redirect('thanks')
+            return redirect('manage_egos_list')
             
     else:
         form = EgoForm()
 
     return render(request, 'ego.html', {'form': form})
 
+@login_required
+def update_ego(request, pk):
+    ego = get_object_or_404(Ego, pk=pk)
```

Comment:
We could do an extra check here and on the delete view if the object is owned by the user:

```
ego = get_object_or_404(Ego, pk=pk, user=request.user)
```

This to avoid users modifying each other's stuff.